libs/MeshKernelApi/src/MeshKernel.cpp

```diff
@@ -95,33 +100,34 @@ namespace meshkernelapi
         }
         catch (const meshkernel::NotImplemented& e)
         {
-            std::memcpy(exceptionMessage, e.what(), sizeof exceptionMessage);
+            CopyExceptionMessage(e.what());
             return MeshKernelApiErrors::NotImplemented;
         }
         catch (const meshkernel::MeshGeometryError& e)
         {
             meshGeometryError = e;
-            std::memcpy(exceptionMessage, e.what(), sizeof exceptionMessage);
+            CopyExceptionMessage(e.what());
             return MeshKernelApiErrors::MeshGeometryError;
```

Comment:
the global meshGeometryError is only assigned in this catch block, should is be assigned in the other catch blocks too?